x-pack/plugins/index_management/common/lib/data_stream_serialization.ts

```diff
@@ -54,3 +54,19 @@ export function deserializeDataStreamList(
 ): DataStream[] {
   return dataStreamsFromEs.map((dataStream) => deserializeDataStream(dataStream));
 }
+
+export const splitSizeAndUnits = (field: string): { size: string; unit: string } => {
```

Comment:
I sort of recall having a helper/lib for this already but I can't seem to find it

Reply:
We have it in ILM, but ILM is only present in stateless so we cant require it from the plugin. I decided to copy it over and added some extra tests at least for now